acm/overlays/moc/infra/managedclustersetbindings/osc.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: cluster.open-cluster-management.io/v1alpha1
+kind: ManagedClusterSetBinding
+metadata:
+  name: osc
+  namespace: argocd
```

Comment:
i just noticed the namespace is only added in this manifest.

Reply:
yep that's intended, the other manifests are cluster-wide.